addons/addon-base-raas/packages/base-raas-services/lib/cfn-templates/cfn-template-service.js

```diff
@@ -48,6 +54,19 @@ class CfnTemplateService extends Service {
 
   async getTemplate(name) {
     const entry = _.find(this.store, ['key', name]);
+    const isAppStreamEnabled = await this.settings.get(settingKeys.isAppStreamEnabled);
```

Comment:
Could we use `settings.getBoolean` instead?

Reply:
updated to use getBoolean